### PR TITLE
Print version on launch

### PR DESCRIPTION
**Pull request by mrkldshv**
Screenshot:
Screen Shot 2023-04-09 at 16 30 28

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -69,7 +70,7 @@ async function main() {
     await inherit()
     return
   }
-  const done = log.timedTask(kleur.bold().bgGreen(' lazyrepo '))
+  const done = log.timedTask(kleur.bold().bgGreen(` lazyrepo v${VERSION}`))
```
Comment (mrkldshv) [inline src/cli.js:73]: @ds300 I'm not entirely sure about this format. Let me know if you have any suggestions on where we should include a version number.

Comment (ds300) [reply src/cli.js:73]: I'm not tied to this green thing with colons in any way, do you want to play around and come up with something different that works better with a version string? Feel free to experiment with color, boldness, backgrounds, whatever.

If that kinda thing is not your bag I'm happy to merge this and maybe tweak it later. Lemme know :)

Comment (mrkldshv) [reply src/cli.js:73]: I'd be happy to experiment with it and provide a few variants. Will let you know when I have something :)

**Comment by ds300** [comment]
Closing this for now, it seems to have stalled, and i added the version number in #41 – there's plenty of room for improvement though if you want to open another PR tweaking the format.

image

**Comment by mrkldshv** [comment]
Sure, I was planning to get back to it after the CLI refactoring, but saw you added version number in one of the PRs.

**Pull request closed without merging**